### DISCUSSION TITLE
Why browser support isn't a project goal?

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [sindresorhus/got#1775] [Q&A] [answered]
As [comparison table](https://github.com/sindresorhus/got#comparison) shows the only feature the project lacks is browser support.
Why is that?

**Comment by szmarczak** [comment, marked as answer]
The environments differ too much. There is another project aimed at browsers which has almost 1:1 interface with Got: https://github.com/sindresorhus/ky

I highly recommend it :)
